dusk/script/__init__.py

```diff
@@ -0,0 +1,84 @@
+import typing
+from dusk.script import internal
+
+from dusk.script.math import *
+from dusk.script.math import __all__ as __math_all__
+
+__all__ = [
+    "stencil",
+    "Edge",
+    "Cell",
+    "Vertex",
+    "K",
+    "Field",
+    "levels_upward",
+    "levels_downward",
+    "sparse",
+    "reduce_over",
+    "mul",
+    "sum_over",
+    "min_over",
+    "max_over",
+] + __math_all__
+
+
+def stencil(stencil: typing.Callable) -> typing.Callable:
+    return stencil
+
+
+class Edge(metaclass=internal.LocationType):
+    pass
+
+
+class Cell(metaclass=internal.LocationType):
+    pass
+
+
+class Vertex(metaclass=internal.LocationType):
+    pass
+
+
+class K:
+    pass
+
+
+class Field:
+    @classmethod
+    def __class_getitem__(cls, *args) -> type:
+        pass
```

Comment:
We actually don't need any of these to parse. There's still two good reasons why we should have them:

1) It helps python's linters (at least some of them). Otherwise, your IDE will constantly tell you that your dusk code is wrong Python. Also, you'll get better autocompletion if they're done well.
2) Without them, we can't import dusk files into Python. So we couldn't write `transpile_and_validate(<<method symbol>>)`. Instead we'd have to do something like `transpile_and_validate("<<dusk file>>", "<<dusk stencil>>")`.

`LocationType` is required as a meta class, so that we can overload the `>` operator for Types (`Vertex`, `Edge` & `Cell`). It's _kinda messy_ to overload the `>` operator for types, but the syntax is much nicer and we can at least satisfy pylint this way.

Reply:
Ok got it! thx!